signup.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>회원가입</title>
+    <link rel="stylesheet" href="css/signup_style.css">
+  </head>
+  <body>
+    <div class="content">
+      <div>
+        <a class="logo" href="index.html"><img alt="logo" src="images/logo.svg"></a>
```

Comment:
이미지 태그의 alt 속성은 해당 이미지에 대한 설명입니다. 기왕이면 어떤 이미지인지 알 수 있게 자세히 적어주는 것이 좋아요.

```suggestion
        <a class="logo" href="index.html"><img alt="linkbrary logo" src="images/logo.svg"></a>
```